py/rackattack/tcp/client.py

```diff
@@ -31,6 +33,13 @@ def allocate(self, requirements, allocationInfo):
             cmd='allocate',
             requirements=jsonableRequirements,
             allocationInfo=allocationInfo.__dict__)
+        return self.getAllocationInstance(requirements, allocationID)
+
+    def allocateExisting(self, requirements, allocationID):
```

Comment:
not private, used here https://github.com/Stratoscale/pyracktest/pull/32/files#diff-1f3d604aca19739700e79a0a6e0b20acR23